os/booting-on-cloudca.md

```diff
@@ -0,0 +1,260 @@
+#cloud-config
+coreos:
+  etcd2:
+    # generate a new token for each unique cluster from https://discovery.etcd.io/new?size=3
+    # specify the initial size of your cluster with ?size=X
+    discovery: https://discovery.etcd.io/<token>
+    # multi-region and multi-cloud deployments need to use $public_ipv4
+    advertise-client-urls: http://$private_ipv4:2379,http://$private_ipv4:4001
+    initial-advertise-peer-urls: http://$private_ipv4:2380
+    # listen on both the official ports and the legacy ports
+    # legacy ports can be omitted if your application doesn't depend on them
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://$private_ipv4:2380
+  units:
+    - name: etcd2.service
+      command: start
+    - name: fleet.service
+      command: start
+    - name: format-ephemeral.service
+      command: start
+      content: |
+        [Unit]
+        Description=Formats the ephemeral drive
+        [Service]
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStart=/usr/sbin/wipefs -f /dev/xvdb
+        ExecStart=/usr/sbin/mkfs.btrfs -f /dev/xvdb
+    - name: var-lib-docker.mount
+      command: start
+      content: |
+        [Unit]
+        Description=Mount ephemeral to /var/lib/docker
+        Requires=format-ephemeral.service
+        After=format-ephemeral.service
+        Before=docker.service
+        [Mount]
+        What=/dev/xvdb
+        Where=/var/lib/docker
+        Type=btrfs
+```
+
+The $public_ipv4 substitution variable is not supported in cloud-config on
+cloud.ca. To use a multi-region cluster, the cluster configuration and the
+port-forwarding must be done manually.
+
+
+### Adding More Machines
+
+To add more instances to the cluster, just launch more with the same
+cloud-config in the same VPC. New instances will join the etcd2 cluster
+automatically.
+
+
+## Launching Instances
+
+Before deploying CoreOS on cloud.ca, you need the following: 
+
+* An account on https://your_organization_name.cloud.ca.
+* Access to an Environment with a running VPC. VPC and subnet creation are not
+  part of the current documentation, refer to cloud.ca online help. 
+* Your public SSH-key added into your cloud.ca environment.
+
+
+### Using cloud.ca admin portal
+
+1. Once logged into your organization; go into Services, select your environment,
+   then click "Add instance":
+<div class="row">
+  <div class="col-lg-8 col-md-10 col-sm-8 col-xs-12">
+    <img src="img/cloudca-addinstance.png" class="screenshot" />
+    <div class="caption">Environment instances inventory</div>
+  </div>
+</div>
+2. Define the instance name, offering type and select the template "CoreOS Stable":
+<div class="row">
+  <div class="col-lg-8 col-md-10 col-sm-8 col-xs-12">
+    <img src="img/cloudca-addinstance_step1.png" class="screenshot" />
+    <div class="caption">New instance details</div>
+  </div>
+</div>
+3. Create a data disk. This volume will be mount in ``/var/lib/docker`` defined
```

Comment:
Create a data volume. This volume will be mounted in `/var/lib/docker`, as defined in the cloud-config excerpt. This step is optional.

(Suggested changes: Use term matching label in UI (data volume). Correct tense (mount*ed*). Clarify what is optional (the whole step rather than the cloud-config excerpt).